xcodeproj/internal/xcschemes/xcschemes.bzl

```diff
@@ -1179,6 +1179,35 @@ Address Sanitizer cannot be used together with Thread Sanitizer.
         ),
     )
 
+def _autogeneration_config(exclude_patterns = None):
+    """Dictionary containing scheme auto-generation configurations
+
+    Args:
+        exclude_patterns: struct returned by `xcschemes.autogeneration_exclude_patterns`
+
+    Returns:
+        A dictionary where keys are strings and the values are lists of strings
+    """
+    d = {}
+    if exclude_patterns:
+        d["exclude_patterns_target_name"] = exclude_patterns.target_name
```

Comment:
Same here about target_names over target_name so it's clearer it's a list

Reply:
Let's discuss here: https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027#discussion_r1603660226